specs/electra/beacon-chain.md

```diff
@@ -1047,59 +1000,6 @@ def process_withdrawals(state: BeaconState, payload: ExecutionPayload) -> None:
         next_validator_index = ValidatorIndex(next_index % len(state.validators))
         state.next_withdrawal_validator_index = next_validator_index
 ```
-
-#### Execution payload
-
-##### Modified `process_execution_payload`
-
-*Note*: The function `process_execution_payload` is modified to use the new `ExecutionPayloadHeader` type.
-
-```python
-def process_execution_payload(state: BeaconState, body: BeaconBlockBody, execution_engine: ExecutionEngine) -> None:
-    payload = body.execution_payload
-
-    # Verify consistency of the parent hash with respect to the previous execution payload header
-    assert payload.parent_hash == state.latest_execution_payload_header.block_hash
-    # Verify prev_randao
-    assert payload.prev_randao == get_randao_mix(state, get_current_epoch(state))
-    # Verify timestamp
-    assert payload.timestamp == compute_timestamp_at_slot(state, state.slot)
-    # Verify commitments are under limit
-    assert len(body.blob_kzg_commitments) <= MAX_BLOBS_PER_BLOCK
-    # Verify the execution payload is valid
-    versioned_hashes = [kzg_commitment_to_versioned_hash(commitment) for commitment in body.blob_kzg_commitments]
-    assert execution_engine.verify_and_notify_new_payload(
-        NewPayloadRequest(
-            execution_payload=payload,
```

Comment:
we would still need to send the `ExecutionLayerRequests` here for verification

otherwise, there is no way to communicate that the requests derived on the EL matched the ones we apply on the CL

Reply:
Yeah good catch. I think you're right.

Reply:
Nice catch! :)